content/en/about/community/partners/index.md

```diff
@@ -27,6 +27,7 @@ Istio benefits from an ecosystem of great partners. Check 'em out and learn how
     {{< company_logo link="https://layer5.io/" logo="./layer5.svg" alt="Layer5" tagline="The service mesh company" >}}
     {{< company_logo link="https://pivotal.io" logo="./pivotal.svg" alt="Pivotal" tagline="The way the future gets built" >}}
     {{< company_logo link="https://www.redhat.com" logo="./redhat.svg" alt="Red Hat" tagline="We make open source technologies for the enterprise" >}}
+    {{< company_logo link="https://www.signalsciences.com" logo="./signalsciences.svg" alt="Signal Sciences" tagline="The world's most trusted web defense" >}}
```

Comment:
Missing image signalsciences.svg

Reply:
Hi Frank - sorry I wasn't sure how to send that over. It's available here - https://www.signalsciences.com/wp-content/uploads/2020/05/signalsciences.svg 
Please let me know if there's another way to share it.

Reply:
@tchang-sigsci You need to put a copy of the image in the [partners directory](https://github.com/istio/istio.io/tree/master/content/en/about/community/partners).

Also notice that the CLA bot is asking you to sign the license agreement.

Reply:
Thanks @frankbu ! I've submitted the logo file [here](https://github.com/istio/istio.io/pull/7311/commits/09dab111878ad65bd1ed0b8fac073e76610bd394). 

I also signed the CLA yesterday in Docusign and ensured that my github email matched the email I used for the CLA. However, I don't see the signed CLA in my list of agreements in the developer portal so maybe there's a lag time?

Reply:
@tchang-sigsci Please include  the logo image to this PR.

Reply:
@frankbu done.